Python/flowgraph.c

```diff
@@ -223,6 +223,15 @@ dump_basicblock(const basicblock *b)
         }
     }
 }
+
+void
+_PyCfgBuilder_DumpGraph(const basicblock *entryblock)
```

Comment:
Nice, I have this same thing stashed locally and kept thinking I should commit it. I have a few other improvements to this debugging code that I'll put in a PR.